CHANGELOG.md

```diff
@@ -10,6 +10,7 @@ The version headers in this history reflect the versions of Apollo Server itself
 > The changes noted within this `vNEXT` section have not been released yet.  New PRs and commits which introduce changes should include an entry in this `vNEXT` section as part of their development.  When a release is being prepared, a new header will be (manually) created below and the the appropriate changes within that release will be moved into the new section.
 
 - `apollo-server-lambda`: Support file uploads on AWS Lambda [Issue #1419](https://github.com/apollographql/apollo-server/issues/1419) [Issue #1703](https://github.com/apollographql/apollo-server/issues/1703) [PR #3926](https://github.com/apollographql/apollo-server/pull/3926)
+- `apollo-tracing`: This package's internal integration with Apollo Server has been switched from using the soon-to-be-deprecated`graphql-extensions` API to using [the request pipeline plugin API](https://www.apollographql.com/docs/apollo-server/integrations/plugins/).  Behavior should remain otherwise the same.  [PR #3991](https://github.com/apollographql/apollo-server/pull/3991)
```

Comment:
```suggestion
- `apollo-tracing`: This package is considered deprecated and — along with its `tracing: Boolean` configuration option on the `ApolloServer` constructor options — will cease to exist in Apollo Server 3.x.  Until that occurs, we've updated the _internal_ integration of this package with Apollo Server itself to use the newer [request pipeline plugin API](https://www.apollographql.com/docs/apollo-server/integrations/plugins/), rather than the _also_ soon-to-be-deprecated-`graphql-extensions` API it previously leveraged.  The behavior should remain otherwise the same.  [PR #3991](https://github.com/apollographql/apollo-server/pull/3991)
```